src/components/shortcuts.js

```diff
@@ -110,6 +110,19 @@ function showContextMenu(card, options = {}) {
         if (playlistId) {
             const elem = dom.parentWithAttribute(card, 'data-playlistitemid');
             item.PlaylistItemId = elem ? elem.getAttribute('data-playlistitemid') : null;
+
+            const itemsContainer = dom.parentWithAttribute(card, 'is', 'emby-itemscontainer');
+            if (itemsContainer) {
+                let index = 0;
+                for (const listItem of itemsContainer.querySelectorAll('.listItem')) {
+                    const playlistItemId = listItem.getAttribute('data-playlistitemid');
+                    if (playlistItemId == item.PlaylistItemId) {
+                        item.PlaylistIndex = index;
+                    }
+                    index++;
+                }
+                item.PlaylistItems = index;
```

Comment:
Let's change this property name. Elsewhere in the code we already use `PlaylistItems` to mean the actual objects in a playlist. Here it's used as a count of the items in the playlist.

I suggest something like `PlaylistItemCount`.

Reply:
Apologies for the delay; moving houses. Pushed a new commit with the changed variable name.